.github/workflows/testPublish.yml

```diff
@@ -0,0 +1,46 @@
+name: Test and publish
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
```

Comment:
should these be pointing to `main`?

Reply:
Yes, there is no `master` branch. This was missed in a copy and paste here.